web/pages/meetings/1st_workshop_2020_at_bryn_mawr/index.md

```diff
@@ -0,0 +1,148 @@
+title:
```

Comment:
```suggestion
title: 1st PlasmaPy Workshop
```

Reply:
I purposely left this meta field blank.  I don't like the heading it creates and the heading provides no functionality anyways.

I have the page heading as the top-level markdown heading.